### DISCUSSION TITLE
redirect static files

### DISCUSSION THREAD

**Discussion by langert** [pimcore/pimcore#17396] [Q&A / Support]
Hi,

with adding redirects to the redirect table seo urls can easily be handled - fine.
https://pimcore.com/docs/pimcore/10.6/Development_Documentation/MVC/Routing_and_URLs/Redirects.html

But how to redirect a static file like /mypath/myfile.js to /mypath/newfile.js?

Aded this to the redirect table, but got "not found"...
Adding the redirect directly to the htaccess file in /public smells like the wrong way...

**Comment by markus-moser** [comment]
You definitely should do this on webserver level.
